neuralmonkey/readers/plain_text_reader.py

```diff
@@ -19,6 +23,38 @@ def reader(files: List[str]) -> Iterable[List[str]]:
     return reader
 
 
+def column_separated_reader(column: int, delimiter: str = "\t",
+                            quotechar: str = None, encoding: str = "utf-8"):
+    """Get reader for delimiter-separated tokenized text.
+    
+    Args:
+        column: number of column to be returned. It starts with 1 for the first
+    """
+    def reader(files: List[str]) -> Iterable[List[str]]:
+        text_reader = get_plain_text_reader(encoding)
+        for line in text_reader(files):
+            f = io.StringIO(' '.join(line))
+            parsed_csv = list(csv.reader(f, delimiter=delimiter,
+                                         quotechar=quotechar,
+                                         skipinitialspace=True))
+            if len(parsed_csv) != 1:
```

Comment:
... tady se sice má kontrolovat, aby to bylo jedna, ale ta hláška se má říct, pokud `len(parsed_csv[0]) <= column` nebo něco takovýho. Takže tuhle hlášku odstěhovat a tahle kontrola (len != 1) zmizí až se bude yieldovat rovnou  z CSV readeru.